frontend/packages/core/src/button.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import type { Meta } from "@storybook/react";
+
+import type { ButtonProps } from './button'
+
+import { Button } from './button';
+
+export default {
+  title: 'Core/Button/Button',
+  component: Button,
+
+} as Meta;
+
+const Template = (props: ButtonProps) => <Button {...props} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  text: "click here",
+};
+
+export const Destructive  = Template.bind({});
+Destructive.args = {
+  ...Default.args,
```

Comment:
You should be able to remove this since we are overriding the only prop from the default args

Reply:
ooh got it, I now see how this works. fixed!